src/layouts/home-assistant.ts

```diff
@@ -21,13 +21,11 @@ import "./home-assistant-main";
 
 const useHash = __DEMO__;
 const curPath = () =>
-  window.decodeURIComponent(
```

Comment:
I added this when we migrated from app-route but I was unable to find this logic in the original package.